packages/sit-onyx/src/components/OnyxAppLayout/OnyxAppLayout.stories.ts

```diff
@@ -0,0 +1,125 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxAppLayout from "./OnyxAppLayout.vue";
+
+/**
+ * Layout component that structures your application.
+ * Includes space for a nav bar, page content, and overlays above the page or the whole app.
+ * Recommended to use on App.vue level of an application.
+ */
+const meta: Meta<typeof OnyxAppLayout> = {
+  title: "layout component/OnyxAppLayout",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxAppLayout,
+    events: [],
+    argTypes: {
+      navBar: {
+        control: { disabled: true },
+      },
+      default: {
+        control: { disabled: true },
+      },
+      pageOverlay: {
+        control: { disabled: true },
+      },
+      appOverlay: {
+        control: { disabled: true },
+      },
+    },
+    // storybook adds 1rem padding. The app layout fills the full available space
+    // so we need to counteract the padding with a negative margin.
+    decorators: [
+      (story) => ({
+        components: { story },
+        template: `
+          <div style="margin: -1rem;" >
+            <story />
+          </div>`,
+      }),
+    ],
+    render: (args) => ({
+      setup: () => ({ args }),
+      components: { OnyxAppLayout },
+      template: getAppTemplate(args.navBarAlignment === "left"),
+    }),
+  }),
+};
+
+export default meta;
+type Story = StoryObj<typeof OnyxAppLayout>;
+
+/** Standard app layout with a nav bar and some content. */
+export const Default = {
+  args: {},
+} satisfies Story;
+
+/** App layout where the nav bar is left aligned. */
+export const LeftNav = {
+  args: {
+    ...Default.args,
+    navBarAlignment: "left",
+  },
+} satisfies Story;
+
+/** Example of an overlay that covers the whole application. */
+export const AppOverlay = {
+  args: {
+    ...Default.args,
+  },
+  render: (args) => ({
+    setup: () => ({ args }),
+    components: { OnyxAppLayout },
+    template: getAppTemplate(
+      args.navBarAlignment === "left",
+      `<template #appOverlay>
+          <div style="background-color: white; 
+                      position: absolute;
+                      inset: 10rem;
+                      min-width: 5rem;
+                      min-height: 1rem;">
+            This is an overlay that covers the whole app.
+          </div>
+        </template>`,
+    ),
+  }),
+} satisfies Story;
+
+/** Example of an overlay that covers the whole page section of an application. */
+export const PageOverlay = {
+  args: {
+    ...Default.args,
+  },
+  render: (args) => ({
+    setup: () => ({ args }),
+    components: { OnyxAppLayout },
+    template: getAppTemplate(
+      args.navBarAlignment === "left",
+      `<template #pageOverlay>
+          <div style="background-color: #ffffffC7;
```

Comment:
I think we can remove the background color, blur looks already fine to me an it works in dark mode too